horace_core/sqw/@DnDBase/compact.m

```diff
@@ -0,0 +1,49 @@
+function dout = compact(din)
+% Squeezes the data range in a dnd object to eliminate empty bins on
+% borders
+%
+% Particularly of use for contracting dnds for plotting
+%
+% If an array of DnDs is provided, this will return a cell array
+% otherwise it will return a DnD scalar
+%
```

Comment:
An equivalent method is provided for the sqw class to compact its dnd sub-objects